src/ssh.c

```diff
@@ -207,6 +225,10 @@ int dcc_ssh_connect(char *ssh_cmd,
 
     i = 0;
     child_argv[i++] = ssh_cmd;
+    for (j=0; j<num_ssh_args; ) {
+        child_argv[i++] = ssh_args[j++];
```

Comment:
Would it not be a more common behaviour to perform the increments to `i` and `j` in the `for` braces rather than inline?  That would also let you use the more effect, although these days not really any more effect in most optimised compilers, pre-increment rather than post.  You could probably more the i = 0 inside the for loop initialisation too as it is related to it.